pkg/microservice/aslan/core/environment/handler/router.go

```diff
@@ -50,6 +50,8 @@ func (*Router) Inject(router *gin.RouterGroup) {
 	commonEnvCfgs := router.Group("envcfgs")
 	{
 		commonEnvCfgs.GET("/:envName/cfg/:objectName", ListCommonEnvCfgHistory)
+		commonEnvCfgs.GET("", ListLatestEnvCfg)
+		commonEnvCfgs.PUT("/:envName/:type/:objectName/sync", SyncEnvResource)
```

Comment:
new APIs？should permission be taken into consideration?

Reply:
there is no need to add permission control, this apis are called by cron